content/influxdb/cloud-dedicated/reference/cli/influxctl/database/create.md

```diff
@@ -0,0 +1,50 @@
+---
+title: influxctl database create
+description: >
+  The `influxctl database create` command creates a new database in an InfluxDB
+  Cloud Dedicated cluster.
+menu:
+  influxdb_cloud_dedicated:
+    parent: influxctl database
+weight: 301
+---
+
+The `influxctl database create` command creates a new database in an InfluxDB
+Cloud Dedicated cluster.
+
+## Usage
+
+```sh
+influxctl database create [--retention-period 0s] {name}
```

Comment:
> influxctl database create [--retention-period 0s] {name}


`{name}` or `<DATABASE_NAME>`?

Just wanna be consistent.

Reply:
> My thinking here is that this is the command signature, not an example.

correct - my formatting was meant as an example of the command signature. Where '[]' indicated optional arguments and '{}' require required arguments.

Looking at a few man pages and the existing [influxdb cli](https://docs.influxdata.com/influxdb/cloud/reference/cli/influx/config/) docs it does look like I should have used `<name>` or `<database_name>` to specify a required argument.

How about we go with `<database_name>` if you want to keep this as a signature?

Reply:
Switched all the arguments to use the `<ARG>` syntax.